spec/calendar.html

```diff
@@ -167,7 +167,6 @@ <h1>CalendarMonth ( _calendar_, _dateLike_ )</h1>
       <emu-alg>
         1. Assert: Type(_calendar_) is Object.
         1. Let _result_ be ? Invoke(_calendar_, *"month"*, « _dateLike_ »).
-        1. If _result_ is *undefined*, throw a *RangeError* exception.
```

Comment:
Elsewhere when this is left implicit, we often add an emu-note saying something like "NOTE: ToPositiveInteger(undefined) will throw a RangeError" or something like that.